js/viz/funnel.d.ts

```diff
@@ -57,13 +58,16 @@ export {
     Palette,
     PaletteExtensionMode,
     TextOverflow,
+    ShiftLabelOverlap,
     WordWrap,
 };
 
 /** @public */
 export type FunnelAlgorithm = 'dynamicHeight' | 'dynamicSlope';
-/** @public */
-export type FunnelLabelOverlap = 'hide' | 'none' | 'shift';
+/**
+ * @deprecated Use ShiftLabelOverlap instead
```

Comment:
please, make it exactly how it was suggested in the appropriate review note

Reply:
done